### DISCUSSION TITLE
[Feature suggestion] 2-layer priority system.

### DISCUSSION THREAD

**Discussion by DShip91** [Raytwo/ARCropolis#326] [Ideas] [answered]
Basically, I think it would be a good idea to allow one mod to take priority over the other for a specific file but no more than that to minimize the risk of crashing.

To provide an example of how this would work, let's say you have two music packs that both edit some of the same songs. The way I envision it is that there would be a setting somewhere in the ARCropolis' config folder to select the files and which mods to put in the priority system (again, no more than two to minimize the risk of crashing).

So for one game file, you could choose which two music packs would take priority over the other rather than having to manually add or remove the files from the mod pack every time you want to disable one or the other. It would basically be a qualty-of-life change.

**Comment by Raytwo** [comment, marked as answer]
Again, no, please stop thinking of overwriting and "priority" as a feature to use and exploit. I'm pretty sure you've brought this up 2 to 3 times already. Not only does this sound like hell to implement without slowing down everything, it'd still open up the door for more user errors.
The conflict handler is there for a good reason and ways to bypass it won't be provided
